src/asmcnc/skavaUI/screen_go.py

```diff
@@ -707,19 +724,19 @@ def _pause_job(self):
 
     POLL_FOR_PAUSE_SCREENS = 0.5
 
-    def raise_pause_screens_if_paused(self, dt=0):
+    def raise_pause_screens_if_paused(self, dt=0, override=False):
         # Ok so 'spindle_shutdown' & the above func needs renaming & refactoring,
         # and the shutdown UI commands need pulling out of serial comms altogether, but that's for another day. 
         # For now, this is enough:
 
-        if  self.m.s.is_job_streaming and \
-            self.m.is_machine_paused and \
-            self.m.reason_for_machine_pause and \
-            self.m.reason_for_machine_pause != "Resuming" and \
-            not str(self.m.state()).startswith('Door:3') and \
-            self.start_or_pause_button_image.source == "./asmcnc/skavaUI/img/pause.png":
+        if (self.m.s.is_job_streaming and
+                self.m.is_machine_paused and
+                self.m.reason_for_machine_pause and
+                self.m.reason_for_machine_pause != "Resuming" and
+                not str(self.m.state()).startswith('Door:3') and
+                self.start_or_pause_button_image.source == "./asmcnc/skavaUI/img/pause.png") or override:
 
-            if self.listen_for_pauses != None: 
+            if self.listen_for_pauses:
```

Comment:
I think there's a reason why I specified this, something to do with how the clock event saves to it. Can't remember, but safest to put it back.